compiler/src/dotty/tools/dotc/transform/TransformWildcards.scala

```diff
@@ -16,6 +17,8 @@ class TransformWildcards extends MiniPhaseTransform with IdentityDenotTransforme
 
   override def phaseName = "transformWildcards"
 
+  override def runsAfter: Set[Class[_ <: Phases.Phase]] = Set(classOf[LiftedClasses])
```

Comment:
Only checked experimentally. It failed if I put it afterward, therefore I added the explicit `runsAfter` to make this evident to anyone that tries to reorder phases.